pkg/virt-controller/services/renderresources.go

```diff
@@ -357,6 +357,14 @@ func GetMemoryOverhead(vmi *v1.VirtualMachineInstance, cpuArch string) *resource
 		overhead.Add(resource.MustParse("53Mi"))
 	}
 
+	// Additional overhead for each interface with Passt binding, that forwards all ports.
+	// More information can be found here: https://bugs.passt.top/show_bug.cgi?id=20
+	for _, net := range vmi.Spec.Domain.Devices.Interfaces {
+		if net.Passt != nil && len(net.Ports) == 0 {
```

Comment:
I'm not sure if you have a way to say that here (I wouldn't spend too long on it anyway as this should be temporary), but if it's IPv4-only or IPv6-only you can halve this.

Reply:
There's no way to limit the binding to only IPv4 or IPv6 in the kubevirt Interface API.

Reply:
And I missed that you already added the link to the ticket there. I could actually read commits before commenting :)

Reply:
No problem. I'll open an issue for us to make sure we revise the memory overhead once the bug is closed.
It's a substantial memory reduction that you're working on, so it's important for us to no forget to take advantage of it.
Thanks for pointing that out!